### DISCUSSION TITLE
Syntax ambiguity with debug operator

### DISCUSSION THREAD

**Discussion by mybearworld** [TodePond/GulfOfMexico#641] [Q&A] [answered]
Take this line:

```js
a ? b : c
```

Is it interpreted as a ternary, or as below?

```js
a? // Debug information about "a"
b: // A label "b"
c //
```

**Comment by TodePond** [comment, marked as answer]
sorry there is no ternary so it's like

```
a?
"b : c"!
```

**Reply by mybearworld** [reply]
Interesting. Does that mean there's also no labels?

**Reply by TodePond** [reply]
labels?

**Reply by mybearworld** [reply]
Oh, never mind, I forgot DreamBerd doesn't have loops, so labels are not needed!

**Comment by circle-gon** [comment]
That's not true-- you can also add labels to the if​ statement.

**Reply by mybearworld** [reply]
Huh. I'm glad I never needed to `break` an `if` before

**Reply by circle-gon** [reply]
Does this still mean that @TodePond 's answer is still correct?

**Reply by mybearworld** [reply]
Wait, how do you use labels with `if` statements? I get an "Uncaught SyntaxError: Illegal break statement" when breaking an `if` statement.
